examples/main/app.js

```diff
@@ -127,15 +118,24 @@ class App extends PureComponent {
   _renderExamples() {
     let index = 1;
     const layers = [];
+    const {activeExamples} = this.state;
+
     for (const categoryName of Object.keys(LAYER_CATEGORIES)) {
       for (const exampleName of Object.keys(LAYER_CATEGORIES[categoryName])) {
 
-        const settings = this.state.activeExamples[exampleName];
+        const settings = activeExamples[exampleName];
         // An example is a function that returns a DeckGL layer instance
-        if (settings) {
-          const example = LAYER_CATEGORIES[categoryName][exampleName];
-          layers.push(this._renderExampleLayer(example, settings, index++));
+        if (!settings) {
```

Comment:
Is the `continue` really adding anything here? It could be better to wrap in an if statement? If nested to deep, maybe an additional function is in order?